extensions/amp-viewer-integration/0.1/messaging/messaging.d.ts

```diff
@@ -0,0 +1,48 @@
+export interface AmpViewerMessage {
```

Comment:
Do we have to export `AmpViewerMessage` and `parseMessage` here? They are on their way to become private and I understand we do not expect their usage outside of the internals of this library?

Reply:
I kind of like to have TypeScript parity, but you're right, we shouldn't encourage dependencies on these two. Removed.